api/src/resources/comment.rs

```diff
@@ -361,6 +363,8 @@ pub struct NewAnnotatedComment {
     pub labelling: Option<NewLabelling>,
     #[serde(skip_serializing_if = "Option::is_none")]
     pub entities: Option<NewEntities>,
+    #[serde(skip_serializing_if = "Option::is_none")]
+    pub audio_path: Option<PathBuf>,
```

Comment:
Stupid question: I assume that this doesn't round-trip? If so, should it go in some clearly denoted subsection like "upload_extras" ?

Reply:
I thought it's similar to the existing `annotations` key -- e.g. at the root level, we support 3 keys
 - `comment` (the actual comment content)
 - `annotations` (labels and entitites annotations, they're uploaded separately to a different route and scoped under a comment)
 -  now `audio_path` (also uploaded separately and scoped under a comment), path to a file that contains the audio content. It's unsatisfactory to have this external reference, but bar inline the content of an audio file in json, we can't do much about it
 
> Stupid question: I assume that this doesn't round-trip?